bedrock/mozorg/urls.py

```diff
@@ -86,7 +86,6 @@
     page("contact/spaces/", "mozorg/contact/spaces/spaces-landing.html"),
     page("contact/spaces/beijing/", "mozorg/contact/spaces/beijing.html"),
     page("contact/spaces/berlin/", "mozorg/contact/spaces/berlin.html"),
-    page("contact/spaces/paris/", "mozorg/contact/spaces/paris.html"),
```

Comment:
Suggestion: we should redirect this URL to `/contact/spaces/` rather than let it 404? (plus add a test for the redirect too).